### DISCUSSION TITLE
Is it possible to run Emulator js by just opening the index html file (Every time I try to, it just say network error Evan if I use different Networks)

### DISCUSSION THREAD

**Discussion by Weman7** [EmulatorJS/EmulatorJS#918] [Q&A]
I don't know if this is possible I have just been using the demo site for about 2-3 years and my school finally blocked it, and I know I can open HTML sites without my school blocking them so I was just wondering if this is possible and if not is there an alternate way to use emulators without running a software because our chromebooks cant do that.

**Comment by ethanaobrien** [comment]
You cannot, because no browser can fetch a `file://` url (security reasons). 

This: https://github.com/EmulatorJS/emulator-button or the code editor: https://emulatorjs.org/editor may fit your needs

**Comment by ghost** [comment]
Fork this repo and host it on your own pages website, or deploy it to a service that supports GitHub repos (like Render or Netlify). If the site gets blocked by content filters, adjust some of the keywords within index.html(most prone to content filters IMO).

**Reply by ethanaobrien** [reply]
https://git.ethanthesleepy.one/ethanaobrien/ejs-allinone-files/releases

**Reply by Weman7** [reply]
My laptop says that JavaScript has been blocked on the site. Does that mean Im cooked?
